scanpy/tests/test_metrics.py

```diff
@@ -77,7 +79,10 @@ def test_consistency(metric, threading):
     all_genes = metric(pbmc, layer="raw")
     first_gene = metric(pbmc, vals=pbmc.obs_vector(pbmc.var_names[0], layer="raw"))
 
-    np.testing.assert_allclose(all_genes[0], first_gene, rtol=1e-9)
+    if Version(numba.__version__) < Version("0.57"):
+        np.testing.assert_allclose(all_genes[0], first_gene, rtol=1e-5)
+    else:
+        np.testing.assert_allclose(all_genes[0], first_gene, rtol=1e-9)
```

Comment:
too much duplication for a one character change

```suggestion
    rtol = 1e-5 if Version(numba.__version__) < Version("0.57") else 1e-9
    np.testing.assert_allclose(all_genes[0], first_gene, rtol=rtol)
```

also please add a comment explaining this

Reply:
A comment like:

```
# use a lower tolerance with older versions of numba
```

?

Personally don't think that adds a lot of info on top of reading the code.

Reply:
Yeah, I guess one can infer that older numba yields a less precise result here, and I guess it’s not super important to know why. But do update the formatting please.